### ISSUE TITLE
[Client][NoSuchElement] Issue with map item

### ISSUE THREAD

**Issue by XakepSDK** [SpongePowered/SpongeForge#1456]
**I am currently running...** 

- SpongeForge version: 1.11.2-2282.6.0.0-BETA-2285
- Forge version: 13.20.0.2282
- Java version: 8
- Operating System: Ubuntu

 - [x] Linux (Ubuntu, Debian, Fedora, etc)
 - [ ] macOS
 - [ ] Windows

- Plugins/Mods: 
InventoryTweaks 1.62+beta.85
jei 1.11.2-4.3.3.266
jer 1.11.2-2.0.6.3.5
optifine 1.11.2 hd u b7
true type font replacement 1.11.2-1.7.3
we cui v2
worldedit 1..11.2-6.1.7
xaeros minimap 1.12.7

**Ways to reproduce**
1. Install spongeforge
2. Join to server
3. Get map
4. Right click

**What is the expected result?**
Map with players position

**What is the current result?**
Crash

```
java.util.NoSuchElementException: No value present
	at java.util.Optional.get(Optional.java:135)
	at net.minecraft.item.ItemMap.redirect$onCreateMap$zgh000(ItemMap.java:580)
	at net.minecraft.item.ItemMap.func_190906_a(ItemMap.java:37)
	at net.minecraft.item.ItemEmptyMap.func_77659_a(SourceFile:17)
	at net.minecraft.item.ItemStack.func_77957_a(ItemStack.java:196)
	at net.minecraft.client.multiplayer.PlayerControllerMP.func_187101_a(PlayerControllerMP.java:496)
	at PlayerControllerOF.func_187101_a(PlayerControllerOF.java:72)
	at net.minecraft.client.Minecraft.func_147121_ag(Minecraft.java:1539)
	at net.minecraft.client.Minecraft.func_184117_aA(Minecraft.java:2179)
	at net.minecraft.client.Minecraft.func_184118_az(Minecraft.java:1956)
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1744)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1056)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:372)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
```

**Comment by phit** [comment]
can confirm, don't even need extra mods just SpongeForge is enough (spongeforge-1.11.2-2393-6.1.0-BETA-2471)
repro: craft map on server, hold and rightclick it, your client will crash
if you join again the map works

client crash and log https://gist.github.com/phit/da9f936861317dcdec963a3ea1367d83
server log https://gist.github.com/phit/bb86941f1de24e177e253903c2af3aea

**Comment by phit** [comment]
can't repro with spongeforge-1.12.2-2529-7.0.0-BETA-2753

**Comment by phit** [comment]
Closing this issue, as of today 1.10.2 and 1.11.2 are not longer supported. If a sufficiently, decent, and tested Pull Request is made to fix this bug/issue, it will still be considered, however, it is NOT to be expected that any bug fixes made for 1.12.2 will be back ported to 1.11.2 or 1.10.2.

If you can reproduce this issue with SpongeForge on Minecraft 1.12.2 and this issue was wrongly closed, please [open a new issue](https://github.com/SpongePowered/SpongeForge/issues/new) with updated information.

**Comment by liach** [comment]
SpongePowered/SpongeCommon#1789 can actually fix this issue hopefully.
